fbfmaproom/fbfmaproom.py

```diff
@@ -310,6 +310,25 @@ def generate_tables(
     # that ever actually happen?
     return main_presentation_df, summary_presentation_df, prob_thresh
 
+def merge_tables(summary, table):
```

Comment:
the way I wrote this is probably pretty pedantic but I wanted to remain agnostic as to which columns need to be filled in. Note that the pandas `concat()` method will *not* work if the set of columns are not identical between every dataframe being concatenated.

Reply:
inner means keep only the columns that exist in both; outer means keep all columns that exist in either. So I would think that an outer join was what you want. In any case, there's no harm in being explicit.

Reply:
Are you sure? Outer is the default per the instructions but when I don't modify the two tables to have the same sets of columns it throws and exception

Reply:
What's the exact error message you get when you try an outer join without adding the missing columns first? It seems to work in this simple case:

```
>>> df1 = pd.DataFrame({'foo': [1, 2], 'bar': [3, 6]})
>>> df2 = pd.DataFrame({'foo': [4, 8, 12]})
>>> pd.concat([df1, df2])
   foo  bar
0    1  3.0
1    2  6.0
0    4  NaN
1    8  NaN
2   12  NaN
```

Reply:
I'm not sure why this behavior doesn't work correctly for combining the summary and data tables. I think we should just keep it as is because the code to work around is fairly minimal.

Reply:
I just commented out lines 315-325 and it worked fine for Ethiopia, all three issue months. Can you share a configuration in which the bug manifests itself?

Reply:
Huh. I just did the same and it worked fine for me too. Maybe it was a bug in a previous version of pandas? (Recall when I first was developing this it was with the environment that had the issues). I guess I'll remove the lines then since I can confirm they definitely aren't needed.

Reply:
Alright, I deleted those lines

Reply:
That seems plausible (pandas bug in previous version).